klass-api/src/main/resources/application-remote-open-search.properties

```diff
@@ -0,0 +1,8 @@
+klass.env.search.elasticsearch.index=klass
+opensearch.username=${OPEN_SEARCH_USERNAME}
+opensearch.url=${OPEN_SEARCH_URI}
+opensearch.password=${OPEN_SEARCH_PASSWORD}
```

Comment:
This pattern generally isn't necessary since all properties can be overridden by environment variables. It would be better to have some kind of sensible default here and then use the in-built environment variable parsing to set the correct values in the nais.yaml files.